samtranslator/model/__init__.py

```diff
@@ -536,6 +552,25 @@ def _construct_tag_list(
         # customer's knowledge.
         return get_tag_list(sam_tag) + get_tag_list(additional_tags) + get_tag_list(tags)
 
+    @staticmethod
+    def propagate_tags(
+        resources: List[Resource], tags: Optional[Dict[str, Any]], propagate_tags: Optional[bool] = False
```

Comment:
We have `tags` in the Resource class, so there is no need to pass them as a parameter and keep this method static.

Reply:
Good point. Going to change this in next PR that includes function/sfn changes.

Reply:
Actually, this method is already a static method, it cannot access any class attributes.